.travis.yml

```diff
@@ -62,6 +62,7 @@ install:
     # Build libcstor for uzfs feature
     - git clone https://github.com/openebs/libcstor.git
     - cd libcstor
+    - if [ ${TRAVIS_BRANCH} == "develop" ]; then git checkout master; else git checkout ${TRAVIS_BRANCH} || git checkout master; fi
```

Comment:
we should probably switch to TRAVIS_TAG for libcstor if provided for cstor. Otherwise everytime we build from the libcstor/master only.